boto3/docs/base.py

```diff
@@ -36,6 +36,6 @@ class NestedDocumenter(BaseDocumenter):
     def __init__(self, resource, root_docs_path):
         super().__init__(resource)
         self._root_docs_path = root_docs_path
-        self._resource_sub_path = self._resource_name
+        self._resource_sub_path = self._resource_name.lower()
         if self._resource_name == self._service_name:
-            self._resource_sub_path = 'ServiceResource'
+            self._resource_sub_path = 'service-resource'
```

Comment:
What prompted the name changes here?

Reply:
For the reasons in my [previous response](https://github.com/boto/boto3/pull/3597#discussion_r1110317266), this had to be 'serviceresource' or 'service-resource'. I thought the latter looked better and also correlated with how the Service Resource section fragment looks `#service-resource`. Either one can be supported.